<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppOnline</class>
 <widget class="QDialog" name="AppOnline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Your Preffered Books</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/icons.qrc">
    <normaloff>:/icons/app_icon</normaloff>:/icons/app_icon</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <item>
    <layout class="QHBoxLayout" name="CentralWidget">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="MainPanel">
       <item>
        <widget class="QLineEdit" name="TxtSearch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Search Songbooks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="LstBooks">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="ProgressLayout">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>99</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="format">
            <string>%p%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="BottomLayout">
         <item>
          <widget class="QLabel" name="LblNothing">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnProceed">
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>PROCEED</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnCancel">
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>CANCEL</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
